src/redteam/constants.ts

```diff
@@ -634,11 +636,13 @@ export const displayNameOverrides: Record<Plugin | Strategy, string> = {
   'harmful:chemical-biological-weapons': 'WMD Content',
   'harmful:child-exploitation': 'Child Exploitation',
   'harmful:copyright-violations': 'IP Violations',
+  'harmful:cybercrime:malicious-code': 'Malicious Code',
```

Comment:
thank you for sorting this. nit, prefer `harmful:cybercrime:malicious-code` after `harmful:cybercrime` - I can check to see if we can update the formatter